src/main/java/org/hisp/dhis/Main.java

```diff
@@ -54,12 +60,24 @@ public static void main( String[] args )
         {
             cache = CacheUtils.deserializeCache();
         }
-        System.out.println( "cache loaded" );
+        System.out.println( "cache loaded from " +  getCachePath() );
 
         LocustSlave locustSlave = LocustSlave.newInstance();
 
         Locust locust = locustSlave.init();
     
         locust.run( new AddTeiTask( 50, cache ) );
     }
+
+    /**
+     * Utility method to generate a single TEI and dump the result in the console (useful for debugging, etc).
+     */
+    private static void generateTei( EntitiesCache cache ) throws IOException {
```

Comment:
Can this be moved somewhere else? I see it's not used anywhere.

Reply:
Removed